lib/mappings.js

```diff
@@ -103,10 +112,18 @@ const convertFuncs = {
 
         }, {});
 
-        Hoek.assert(badChildren.length === 0, `Joi Schema type(s) "${badChildren}" not supported.`);
+        Hoek.assert(badChildren.length === 0, `Joi Schema type(s) "${badChildren}" not supported. Please fix mappings.js`);
```

Comment:
Is this a user-facing error?  I wonder if `Please fix mappings.js` could be confusing to a user.